scripts/stylish.sh

```diff
@@ -6,7 +6,7 @@ export TOP=${TOP:-$(git rev-parse --show-toplevel)}
 
 source $TOP/scripts/run-on-haskell.include.sh
 
-stack build stylish-haskell
+stack install stylish-haskell
 export PATH=$(stack path --bin-path)
```

Comment:
Setting `PATH` should no longer be necessary, if we are installing `stylish-haskell` into the `local-bin-path`.